virttest/qemu_vm.py

```diff
@@ -4300,11 +4300,19 @@ def migrate(self, timeout=virt_vm.BaseVM.MIGRATE_TIMEOUT, protocol="tcp",
             # If we're doing remote migration and it's completed successfully,
             # self points to a dead VM object
             if not not_wait_for_migration:
-                if self.is_alive() and self.is_paused():
+                if self.is_alive():
                     # For short period of time the status can be "inmigrate"
                     # for example when using external program
                     # (qemu commit fe823b6f87b2ebedd692ca480ceb9693439d816e)
-                    self.resume(60)
+                    # resume vm during "inmigrate" status can't work, wait until
+                    # exit "inmigrate" status and resume vm if needed.
+                    if utils_misc.wait_for(lambda: not self.monitor.verify_status('inmigrate'),
+                                           timeout=120, step=0.1):
+                        if self.is_paused():
+                            self.resume()
```

Comment:
do we need timeout for this resume?

Reply:
@xutian Thanks, i think it is no need. resume timeout was involved by commit 61dcaa106b5e6a21c3fa7931a35f20374112ff8c, it aim to send "cont" one time then wait for the vm status change from inmigrate to running.  But in fact, send "cont" while vm status is inmigrate, and with "-S",  it can not resume the vm. 
With this patch, it will send 'cont' after vm exit 'inmigrate' status. no need to add timeout.